Source/GameActor.cpp

```diff
@@ -46,18 +46,19 @@ void GameActorImage::render(GameActorInstance* gameActorInstance, Renderer2D* re
 	}
 	renderer->setBlendMode(blendMode());
 
-	Texture* texture = gameActorInstance->gameActor()->m_Textures[textureIndex()];
-	//GraphicsBuffer* vertexBuffer = gameActorInstance->gameActor()->m_VertexBuffer;
-	//GraphicsBuffer* skinnedVertexBuffer = gameActorInstance->gameActor()->m_SkinnedVertexBuffer;
-	GraphicsBuffer* indexBuffer = gameActorInstance->gameActor()->m_IndexBuffer;
```

Comment:
This is getting a single index buffer stored on the whole main actor (multiple instances will use this same single index buffer for drawing). This is nice as it causes less pipeline stalls to switch the active index buffer, but requires offsetting within that index buffer like we used to.